jobconfig.toml

```diff
@@ -169,20 +171,20 @@
 [JANITOR]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 1
+"Spawn Positions" = 2
```

Comment:
northstar/delta support 4, meta/biodome/icebox only 1. Let's just do this, I guess.

Reply:
Reduced spawn position amount to 1 for janitor in next commit; But will see what happens.